django_ai_assistant/helpers/assistants.py

```diff
@@ -462,7 +519,20 @@ def tool_selector(state: AgentState):
             return "continue"
 
         def record_response(state: AgentState):
-            return {"output": state["messages"][-1].content}
+            if self.structured_output:
+                llm_with_structured_output = self.get_structured_output_llm()
+                response = llm_with_structured_output.invoke(
+                    [
+                        *state["messages"],
+                        SystemMessage(
+                            content="Use the information gathered in the conversation to answer."
```

Comment:
What exactly is this system message at the end? Is that the recommended way to get a final JSON output?
Also, maybe mention JSON here in this final message?

Reply:
But why this is a SystemMessage? Are you following any pattern you saw in a tutorial?
I couldn't find much info online about multiple system messages. Why not a user message here?

Reply:
Not following any patterns. Just though it made more sense as as system message. We could as well just don't pass any message.

Reply:
Changed this to HumanMessage for compatibility with Antropic, removing it also breaks the model, it expects a human message before answering